synapse/tests/common.py

```diff
@@ -25,6 +28,14 @@
 
 class TooFewEvents(Exception): pass
 
+# Py2/3 SSL Exception Compat
+if s_compat.version >= (3, 0, 0):
```

Comment:
Maybe this means we need our own ssl exception to unify the discrepancy?  ( maybe even SynSockErr and SynSslErr ? )